net-analyzer/bro/bro-2.5.0.ebuild

```diff
@@ -0,0 +1,75 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_REQ_USE="sqlite"
+PYTHON_COMPAT=( python{2_7,3_{4,5,6}} )
+inherit cmake-utils python-single-r1
+
+BRO_VER="2.5"
+
+DESCRIPTION="The Bro Network Security Monitor"
+HOMEPAGE="https://www.bro.org"
+SRC_URI="https://www.bro.org/downloads/${PN}-${BRO_VER}.tar.gz"
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="-broccoli +broctl +broker curl debug geoip ipv6 jemalloc +python tcmalloc static-libs +tools"
+
+RDEPEND="app-shells/bash:0
+	dev-libs/openssl:0
+	net-analyzer/ipsumdump
+	net-dns/bind-tools
+	net-libs/libpcap
+	sys-libs/zlib
+	broker? ( =dev-libs/actor-framework-0.14* )
+	broctl? ( virtual/mta )
+	curl? ( net-misc/curl )
+	geoip? ( dev-libs/geoip )
+	ipv6? ( net-analyzer/ipsumdump[ipv6] )
+	jemalloc? ( dev-libs/jemalloc )
+	python? ( ${PYTHON_DEPS} )
+	tcmalloc? ( dev-util/google-perftools )"
+DEPEND="${RDEPEND}
+	>=dev-lang/swig-3.0.3
+	dev-lang/perl"
+
+REQUIRED_USE="broker? ( python )
+	broctl? ( python )
+	python? ( ${PYTHON_REQUIRED_USE} )"
+PATCHES=( "${FILESDIR}/bro-2.4.1-remove-unnecessary-remove.patch" )
+
+S="${WORKDIR}/${PN}-${BRO_VER}"
+
+pkg_setup() {
+	use python && python-single-r1_pkg_setup
+}
+
+src_configure() {
+	local mycmakeargs=(
+		-DENABLE_DEBUG=$(usex debug true false)
```

Comment:
Is there any specific reason you're overriding the values? Unless upstream does something silly, the default yes/no works just fine for cmake.